### PR TITLE
Change forwarding of stdout from eclipse to .OUT

### PR DESCRIPTION
**Pull request by eivindsm**
… to be compatible with eclrun. A symlink is added .LOG -> .OUT

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: src/subscript/legacy/runeclipse [hunk 1]
```diff
@@ -218,6 +218,18 @@ if ( !-f $exe64 ) {
     $testversion = 1;
 }
 
+#
+# Forward stdout to rootname.OUT for versions 2019.3 and younger for consitancy with eclrun
```
Comment (berland) [inline src/subscript/legacy/runeclipse:222]: typo: consistency

**Comment by berland** [comment]
Looks ok.

First  line in git commit  should be limited to 50 chars (git commit lines are parsed and used for release notes)

File: src/subscript/legacy/runeclipse (hunk 1 above)
Comment (eivindsm) [reply src/subscript/legacy/runeclipse]: Fixed

**Comment by berland** [comment]
Squash before merge or squash while merge? (the latter skips review of commit message..)

**Review by berland — APPROVED** [review]
OK. Squash when merge, and fine-tune the commit message.

**Pull request merged**